### PR TITLE
multiple empty bullets fixed & user story image resized

### PR DESCRIPTION
**Pull request by bessii**
Fixes issue #825 and #837 

Changes proposed in this pull request:

- i added a front matter section with the title of the page on each of the pages in admin-guide directory which corrected the multiple empty bullets

- i did same for install-guide, intro-admin, intro-user, how-to, self-hosted and vmm-guide directories

- resized the image in Alter way case study

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @bessii on yourself to sign)

This pull request needs review by: (please @jasonbrooks  the reviewer if relevant)

### CODE REVIEW

**Comment by mykaul** [comment]
@JohnMarksRH - can you look at this?

**Comment by jasonbrooks** [comment]
These LGTM, but should they be submitted in separate PRs?

**Comment by duck-rh** [comment]
So, you're trying to solve two totally unrelated tickets in one single PR, this is not the policy, so please split.

Also you're fixing typos or small mistakes along the way,which is good, but also should be in a separate PR.

In source/community/user-stories/alter-way-case-study.html.md I see a space added to a class, why so? I guess it is unwanted change, so please remove.

When this is properly separated we can get to the changes review per see. Thanks.

File: source/develop/developer-guide/vdsm/hook/hugepages.html.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ The hugepages vdsm hook will receive hugepages=512 and will preserve 512 huge pa
 
 This is useful for VMs running large memory loads, so that memory page fragmentation is lowered and the VM can access larger memory chunks at a time. The downside is more potential for wasted RAM.
 
-How this works: The hook will add pages: sysctl vm.nr_hugepages=512 and will add add the following xml in domain\\devices:
+How this works: The hook will add pages: sysctl vm.nr_hugepages=512 and will add the following xml in domain\\devices:
 
-`   `<memoryBacking>
-`       `<hugepages/>
-`   `</memoryBacking>
+`   <memoryBacking>  `
```
Comment (mykaul) [inline source/develop/developer-guide/vdsm/hook/hugepages.html.md]: The change doesn't look right:
   <memoryBacking>      <hugepages/>     </memoryBacking>

It probably ought to be in separate lines.

**Comment by bessii** [comment]
thanks a lot. I will create branches, work on the files again and submit my changes in separate pull requests.

**Comment by JohnMarksRH** [comment]
@bessii Hi, thanks for resizing the image in the Alter Way case study. Do you have someway to demonstrate that it's fixed? Some type of staging environment? That would be really helpful.

**Pull request closed without merging**